### ISSUE TITLE
Partial MySQL connection

### ISSUE THREAD

**Issue by CupidonSauce173** [poggit-orphanage/FactionsPro_1.4#14]
This is an idea, you should make factions to be able to connect to a MySQL server. But only the faction of a player, not the plot or anything else.

**Comment by SOF3** [comment]
I don't think this is possible under a proper SQL structure. It will have to require saving data twice.

**Comment by SOF3** [comment]
What is your motivation? Perhaps we can work out a better alternative for you.

**Comment by SOF3** [comment]
Oops, I forgot to unlock the issue

**Comment by CupidonSauce173** [comment]
What would be a good alternative ?

**Comment by TheShinPin** [comment]
@SOF3 We are working on making a network of servers, and it would be weird to have someone own a faction on 1 server, and not on the other. We want to connect our factions together across servers, giving a better and higher quality.

**Comment by SOF3** [comment]
It is really strange to have the faction membership in MySQL and faction chunks in SQLite3. It is not impossible, but it kills the point of using foreign keys. (Does FactionsPro use it anyway? Don't use FactionsPro; it sucks!)
